pkg/challenges/challenges_test.go

```diff
@@ -346,6 +490,62 @@ func Test_isURISubjectAllowed(t *testing.T) {
 	}
 }
 
+// reflect hack because "claims" field is unexported by oidc IDToken
+// https://github.com/coreos/go-oidc/pull/329
+func updateIDToken(idToken *oidc.IDToken, fieldName string, data []byte) {
+	val := reflect.Indirect(reflect.ValueOf(idToken))
+	member := val.FieldByName(fieldName)
+	pointer := unsafe.Pointer(member.UnsafeAddr())
+	realPointer := (*[]byte)(pointer)
+	*realPointer = data
+}
+
+func TestEmail(t *testing.T) {
```

Comment:
nit: can we call this `TestEmailWithClaims` or something, since the test is primarily focused on that part of the `email` function?